W3ChampionsStatisticService/PlayerStats/GameLengthForPlayerStatistics/PlayerGameLengthStat.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections.Generic;
+using System.Linq;
+using W3ChampionsStatisticService.W3ChampionsStats.GameLengths;
+
+namespace W3ChampionsStatisticService.PlayerStats.GameLengthForPlayerStatistics;
+
+public class PlayerGameLengthStat
+{
+    // the dictionary key is the interval start: 0, 30, 60, 90, 120..
+    // the dictionary value is the number of games
+    public Dictionary<string, int> Lengths { get; set; }
+
+    public void Record(int duration)
+    {
+        var groupInterval = 60;
+        var maxGroupValue = 60;
+        var group = (int)duration / groupInterval;
```

Comment:
Why is a cast to int necessary?

Reply:
To get a Integer instead of float

Reply:
Oh okay. I thought it was just casting the duration.